src/content/overview/architecture/_index.md

```diff
@@ -11,7 +11,77 @@ user_questions:
+
+## Platform architecture
+
+![Platform architecture](./platform-architecture.png)
+
+The platform architecture consists of the following layers:
+
+- Interface layer: offers a set of templates, APIs, and tools that developers can use to interact with the platform.
+
+- Capabilities layer: provides a set of capabilities to manage the lifecycle of applications, including deploying, scaling, and monitoring applications.
+
+- Infrastructure layer: responsible for managing the underlying infrastructure, including the cloud provider resources or on-premises infrastructure. To abstract the underlying infrastructure, the platform uses the `Kubernetes`.
+
+Along these layers, Giant Swarm provides a real support channel, incident management, and good documentation to help you get the most out of the platform. Additionally, you have a dedicated account engineer to help creating a common roadmap and to provide guidance on how to use the platform.
+
+## Platform API
+
+The Giant Swarm platform is build on top of `Kubernetes` thanks to [`Cluster API`](https://cluster-api.sigs.k8s.io/), a open source  project that standardize the cluster lifecycle management across different cloud providers or on-premises infrastructure.
+
+In `Cluster API`, there is a distinction between the management cluster and the workload clusters. The management cluster is responsible for managing the lifecycle of the workload clusters. The workload clusters are the clusters where you deploy your applications.
+
+In Giant Swarm, the management cluster is also used for enhancing the platform capabilities, such as monitoring, logging, and alerting. The customer platform team uses the management cluster to configure which capabilities are available to the developers, and even to create new capabilities.
+
+Most often, you have a single management cluster that manages multiple workload clusters, but in case of running infrastructure in multiple regions, you can have multiple management clusters. The platform API isn't more than the `Kubernetes` management cluster API, but with some additional resources that allow you to manage the lifecycle of your workloads across multiple clusters or inspect the application metrics of your fleet of clusters.
+
+[Learn how to access the platform API]({{< relref "/getting-started/access-to-platform-api" >}}).
+
+## GitOps
+
+Giant Swarm uses GitOps as first-class citizen to manage the lifecycle of cluster and applications. It's modern operational framework which allow managing our resources using `Git` repositories as the single source of truth. In the platform, GitOps ensures that all changes are version-controlled, auditable, and easily reversible. By leveraging Git’s robust workflows, teams can automate deployments and rollbacks while maintaining consistency and reliability across environments. This capability empowers developers to implement continuous delivery practices, facilitating faster and more secure releases.
```

Comment:
```suggestion
Giant Swarm uses GitOps as a first-class citizen to manage the lifecycle of clusters and applications. It's a modern operational framework which allows for managing our resources using `Git` repositories as the single source of truth. In the platform, GitOps ensures that all changes are version-controlled, auditable, and easily reversible. By leveraging Git’s robust workflows, teams can automate deployments and rollbacks while maintaining consistency and reliability across environments. This capability empowers developers to implement continuous delivery practices, facilitating faster and more secure releases.
```

Reply:
I think this is way too long of a description, but doesn't explain how GS "uses GitOps" – we don't enforce it, so the potential customer should be clear here what to expect. I'd rather describe GitOps as built-in capability instead of explaining the concept at length. I'm thinking of something like this: "[... explanation what GitOps is in one sentence ...] We pre-install Flux and you can use it. Only connect your Git control repo and you're good to go." (must be reworded, of course).